designer/server/src/routes/forms/api.js

```diff
@@ -33,16 +33,17 @@ export default [
         parse: true
       },
       async handler(request, h) {
-        const { id } = request.params
+        const { auth, params, payload } = request
+        const { id } = params
+        const author = forms.getAuthor(auth.credentials)
```

Comment:
Do you mind me saying it feels a bit overly-destructured?

E.g. I quite like seeing `params.id` to know where it comes from

```suggestion
        const author = forms.getAuthor(auth.credentials)
```

Reply:
All the handlers do is basically destructring and passing along data.
I find this reads better.

Also, it "scales" better IMO:

E.g.Imagine a route starts out like this:

```
const { params, payload } = request
const { id } = params
const author = forms.getAuthor(auth.credentials)
```

Then sometime later, a new parameter gets added, and the whole route gets authenticated. It's easy to simply extend the destructring:

```
const { auth, params, payload } = request
const { id, slug } = params
```